control-plane/subcommand/webhook-cert-manager/command.go

```diff
@@ -179,13 +177,14 @@ func (c *Command) Run(args []string) int {
 				Expiry: expiry,
 			}
 		}
+
+		certCh := make(chan cert.MetaBundle)
 		certNotify := &cert.Notify{Source: certSource, Ch: certCh, WebhookConfigName: config.Name, SecretName: config.SecretName, SecretNamespace: config.SecretNamespace}
 		notifiers = append(notifiers, certNotify)
 		go certNotify.Start(ctx)
+		go c.certWatcher(ctx, certCh, c.clientset, c.logger)
 	}
 
```

Comment:
The issue is on line 203. `bundle` is always set to the last bundle that came out of `ch`. Then we wake up every `defaultRetryDuration` to check things but we're only checking one `bundle`.